src/selectors/contents.js

```diff
@@ -1,21 +1,39 @@
 import { createSelector } from 'reselect'
 import { first, uniq } from 'lodash'
 
-const getAugmentedContents = (contents, favorites, history) =>
-  contents.map(content => ({
-    ...content,
-    isFavorite: favorites[content.id],
-    lastPlayed: history[content.id],
-  }))
+const getAugmentedContent = (content, favorites) => ({
+  ...content,
+  isFavorite: favorites[content.id],
```

Comment:
`isFavorite: favorites[content.id] || false,`

I like to really have booleans :)